src/bundle/Resources/public/js/scripts/core/tag.view.select.js

```diff
@@ -88,6 +88,8 @@
         }
 
         removeItems(items) {
+            items = items.map(String);
```

Comment:
I think it would be better if you assign it to new variables, so that you won't be modifying function argument, as it can be potentially dangerous, maybe sth like
```suggestion
            const stringifiedItems = items.map(String);
```